.github/workflows/sanity.yml

```diff
@@ -29,6 +29,9 @@ jobs:
       run: make fmt
     - name: Vet
       run: make vet
+    - name: Verify go modules are properly up-to-date
+      run: |
+        make mod-tidy mod-verify && git diff --exit-code
```

Comment:
Can you explain why do we need `git diff --exit-code` here? As far as I understood `make mod-tidy mod-verify` returns proper exit code, no?

Reply:
mod-tidy / go mod tidy exits with a 0 status on success including cases where it needed to update `go.{mod,sum}` but succeeded in doing so. In contrast, here we want to make sure that `go.{mod,sum}` was already up-to-date, or IOW, there was no need to touch and modify them. `git diff --exit-code` is there to make sure that any (successful) changes made to these files are considered an error and not a success.

Reply:
Thank you. lgtm then.